tools/test/test262/test262.py

```diff
@@ -319,9 +320,16 @@ def GetDriverSource(self):
 
   def GetSource(self):
     if self.IsRaw():
-        return self.test
-
-    source = self.test
+        return self.source
+
+    source = self.source
+    # if test is hashbang test
+    if len(source) > 1 and (source[0] == '#' or source[0] == '\\'):
+      lines = source.splitlines()
+      if self.strict_mode:
+        lines.insert(1, 'var strict_mode = true;')
```

Comment:
Isn't your original intention to insert it into the zero index?

Reply:
Yes. 1 is intended number